.travis.yml

```diff
@@ -1,15 +1,21 @@
 language: elixir
 elixir:
-  - 1.7.4
-  - 1.8.1
+  - 1.8.2
+  - 1.9.1
 otp_release:
-  - 20.3
+  - 21.3
+  - 22.0
 env:
   - ""
   - WALLABY_DRIVER=phantom
   - WALLABY_DRIVER=chrome
-  - WALLABY_DRIVER=selenium WALLABY_SELENIUM_VERSION=3
-  - WALLABY_DRIVER=selenium WALLABY_SELENIUM_VERSION=2
+  - WALLABY_DRIVER=selenium
+matrix:
+  exclude:
+  - elixir: 1.8.2
+    otp_release: 22.0
+  - elixir: 1.9.1
+    otp_release: 21.3
```

Comment:
I think I'm okay with leaving these as part of the matrix.

Reply:
I excluded these as suggested by @keathley [here](https://github.com/keathley/wallaby/pull/443#issuecomment-514198815), and I prefer less waiting :)